### ISSUE TITLE
How to get the better image feature extraction for similarity check with model pre-training?

### ISSUE THREAD

**Issue by khpanb** [clint-kristopher-morris/DINO_concise#3]
Hi thanks for this nice code tutorial and explained. i'm trying to use the custom dataset for the image similarity check from feature extraction with dino model. i guess what i need to do is to create the dataset with the same structure as imagenet but without the labels then let model to train it by itself with self-supervising learning from scratch to reduce the loss. Or maybe should i do the fine-tune on the pre-training model ? Most of tuning discussion i found all need many labeled data (i assume the purpose is mainly for downstream tasks like classification or segmentation...etc). Not sure which way are really helpful for the image feature extraction and similarity check.  Do you have any suggestion? thank you!
